packages/access-api/test/provider-add.test.js

```diff
@@ -97,6 +99,95 @@ for (const accessApiVariant of /** @type {const} */ ([
       })
     })
   })
+
+  if (
+    ['provider/add', 'access/delegate'].every((f) =>
+      accessApiVariant.features.has(f)
+    )
+  ) {
+    it('provider/add allows for access/delegate', async () => {
+      const space = await principal.ed25519.generate()
+      const agent = await accessApiVariant.issuer
+      const service = await accessApiVariant.audience
+      const accountDid = /** @type {const} */ ('did:mailto:example.com:foo')
+
+      const accountAuthorizesAgentClaim = await ucanto.delegate({
+        issuer: principal.Absentee.from({ id: accountDid }),
+        audience: agent,
+        capabilities: [
+          {
+            with: 'ucan:*',
+            can: '*',
+          },
+        ],
+      })
+      const serviceAttestsThatAccountAuthorizesAgent = await ucanto.delegate({
+        issuer: service,
+        audience: agent,
+        capabilities: [
+          {
+            with: service.did(),
+            can: 'ucan/attest',
+            nb: { proof: accountAuthorizesAgentClaim.cid },
+          },
+        ],
+      })
+      const sessionProofs = [
+        accountAuthorizesAgentClaim,
+        serviceAttestsThatAccountAuthorizesAgent,
+      ]
+      const addStorageProvider = await ucanto
+        .invoke({
+          issuer: agent,
+          audience: service,
+          capability: {
+            can: 'provider/add',
+            with: accountDid,
+            nb: {
+              provider: 'did:web:web3.storage:providers:w3up-alpha',
+              consumer: space.did(),
+            },
+          },
+          proofs: [...sessionProofs],
+        })
+        .delegate()
+      const addStorageProviderResult = await accessApiVariant.invoke(
+        addStorageProvider
+      )
+      assertNotError(addStorageProviderResult)
+
+      // storage provider added. So we should be able to delegate now
+      const accessDelegate = await ucanto
+        .invoke({
+          issuer: agent,
+          audience: service,
+          capability: {
+            can: 'access/delegate',
+            with: space.did(),
+            nb: {
+              delegations: {},
```

Comment:
Is this supposed to be empty ? because if it is you're not delegating anything here.

Reply:
yes, the specific delegation isn't important. Just want to show that the `access/delegate` invocation is accepted sans error